internal/configs/config.go

```diff
@@ -1018,3 +1045,11 @@ func (c *Config) CheckCoreVersionRequirements() hcl.Diagnostics {
 
 	return diags
 }
+
+// ImpliedProviderType returns true if the given local provider configuration
+// is not explicitly declared in the required_providers block of this module or
+// any of its descendants. This means the underlying "type" of the provider
+// has been implied to be a Hashicorp internal provider.
+func (c *Config) ImpliedProviderType(config addrs.LocalProviderConfig) bool {
+	panic("implement me")
```

Comment:
Is this meant to stay here?

Reply:
No, that was left over from an abandoned approach 😅